test/test_files/tinysnb/function/arithmetic.test

```diff
@@ -332,3 +332,42 @@
 -2
 -100
 7
+
+-NAME BitwiseAndFunctionTest
+-QUERY MATCH (p:person) RETURN p.age & 21
+---- 8
+1
+20
+5
+20
+20
+17
+0
+17
+
+-NAME BitwiseOrFunctionTest
+-QUERY MATCH (o:organisation) RETURN o.score | 35
+---- 3
+-1
+-65
+39
+
+-NAME BitshiftLeftFunctionTest
+-QUERY MATCH (p:person) RETURN p.age << 2
+---- 8
+140
+120
+180
+80
+80
+100
+160
+332
+
+-NAME BitshiftRightFunctionTest
+-QUERY MATCH (o:organisation) RETURN o.orgCode >> 3
```

Comment:
I would also add tests to check if the numerical operator precedence follows the [cpp standard](https://en.cppreference.com/w/c/language/operator_precedence) (I don't think SQL has a standard on this).

e.g. a + 1 & 1 should be interpreted as (a + 1) & 1 since addition has a higher precedence

Reply:
Done in latest commit